validator/src/commands/run/args.rs

```diff
@@ -1640,4 +1641,32 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
                 May get stuck if the leader used is different from others.",
             ),
     )
+    .arg(
+        Arg::with_name("retransmit_xdp_interface")
+            .hidden(hidden_unless_forced())
```

Comment:
what is the point of hiding this?

Reply:
> what is the point of hiding this?

@alexpyattaev this cli arg is experimental, right? We do not want flags like this showing up to every validator operator. We need some level of hiding so you have to find the guns to aim at your feet.

Reply:
Yes its experimental and even if they find it it somehow it would take a few extra steps to actually run with XDP enabled. This particular gun is mostly disassembled at the moment =)